### PR TITLE
Add recruiter field, added another func in Jobs, created migration and tests

### PR DESCRIPTION
**Pull request by yaelerdr**
- Added recruiter as FK to jobboard/Job
- Created new migration (saved as 0005)
- Added a new function that returns all jobs posted by a specific recruiter ("get_jobs_by_recruiter_id")
- Added test for that function
- Changes recruiter's test - added recruiters saved instances in where they were matched to the expected result

### CODE REVIEW

File: jobboard/models.py
Comment (YarinBou) [inline jobboard/models.py:80]: lgtm

File: jobboard/models.py
Comment (YarinBou) [inline jobboard/models.py:108]: I think that a better description will be based on the recruiter's name and maybe even pass the send the recruiter himself as an object and comparing with their id because can be two recruiter's with the same name. you can look at what I did in the job_application model.

File: jobboard/tests.py
Comment (YarinBou) [inline jobboard/tests.py:28]: lgtm

File: jobboard/models.py
Comment (yaelerdr) [reply jobboard/models.py:108]: I agree on that one.
The only problem is that I don't think the function can get a Recruiter instance as an argument..
What do you say of letting it get recruiter_id as an input?

Comment (guyy13) [reply jobboard/models.py:108]: It can get a recruiter instance as an argument

**Comment by guyy13** [comment]
We talked about adding a major field to the Job model.
Maybe wait until #52 is merged and then add a major field based on the Major text choices model in Student.

File: jobboard/migrations/0002_add_example_data.py
Comment (guy9050) [inline jobboard/migrations/0002_add_example_data.py:7]: what's the reason that you add this dependency?

File: jobboard/models.py
Comment (guy9050) [inline jobboard/models.py]: why do you change this?

Comment (yaelerdr) [reply jobboard/models.py]: It was auto changed for some reason. I'll change it back to what it was

File: jobboard/migrations/0002_add_example_data.py
Comment (yaelerdr) [reply jobboard/migrations/0002_add_example_data.py:7]: I added this because 0005 is the migration file I created after adding recruiter as new field. Here, I create job objects with non-null recruiters, so 0005 is a dependency of it

**Comment by eldaraharonson** [comment]
> I think that pytest error might happen because the tests of recruiter are fixture tests, and the tests for job are'nt. As to that, I created an instances of recruiter, and saved it to the db.
> 
> It probably fails because of something with the tests of recruiter(?)
> 
> I have tried to add recruiter/test.py to the dependencies of 0002, but the test fails (without it, it passes) :\

I think you need to consider the new recruiters you created in the recruiter test functions because they functions will probably be accessing it in the class functions.

File: jobboard/models.py
Comment (guy9050) [reply jobboard/models.py:108]: @yaelerdr ?

Comment (yaelerdr) [reply jobboard/models.py:108]: Changed that to be a recruiter instance @guy9050

**Review by guyy13 — APPROVED** [review]

**Review by eldaraharonson — APPROVED** [review]

**Review by YarinBou — APPROVED** [review]

**Comment by guy9050** [comment]
please squash the commits

**Comment by yaelerdr** [comment]
> please squash the commits

@guy9050 done :)

**Review by benmenachem — APPROVED** [review]

**Comment by guy9050** [comment]
I will merge this PR this time, but basically it's too big - next time please divide it to several PR's.

**Pull request merged**
